### PR TITLE
Forntend stock data

### PR DESCRIPTION
**Pull request by LuchoStoymirov**
image

### CODE REVIEW

File: TypeScript/AI_Finance/src/Components/StockData/StockData.tsx [hunk 1]
```diff
@@ -2,33 +2,36 @@ import style from "./StockData.module.css";
...
-      try {
-        setData(await apiService.get());
-      } catch (error) {
-        // eslint-disable-next-line no-console
-        console.log(error);
-        throw error;
-      }
-    })();
-  }, []);
+  // useEffect(() => {
```
Comment (betinabp) [inline TypeScript/AI_Finance/src/Components/StockData/StockData.tsx:10]: add //TODO: implement when BE is ready comment above the useEffect

File: TypeScript/AI_Finance/src/Components/StockData/StockData.module.css [hunk 2]
```diff
@@ -1,17 +1,43 @@
...
\ No newline at end of file
+  .title{
+    margin: 0;
+  }
+
+  .titlebox{
+    margin: 0;
+    margin-bottom: 10px;
+    padding: 10px;
+    padding-left: 7px;
```
Comment (betinabp) [inline TypeScript/AI_Finance/src/Components/StockData/StockData.module.css:39]: Use only even numbers.

File: TypeScript/AI_Finance/src/Components/StockData/StockData.module.css [hunk 3]
```diff
@@ -1,17 +1,43 @@
...
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+  padding-left: 55px;
+  background-color: var(--dataRowBlue);
+  border-bottom: 1px solid var(--lightBlue);
+  border-bottom-left-radius: 5px;
+  border-bottom-right-radius: 5px;
+  margin-bottom: 15px;
+}
```
Comment (betinabp) [inline TypeScript/AI_Finance/src/Components/StockData/StockData.module.css:24]: Use only even numbers inside this class.

File: TypeScript/AI_Finance/src/Components/DataRow/DataRow.tsx [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import { Grid } from "@mui/material";
 import style from "./DataRow.module.css";
 
+
```
Comment (betinabp) [inline TypeScript/AI_Finance/src/Components/DataRow/DataRow.tsx:4]: remove empty line

File: TypeScript/AI_Finance/src/Components/StockData/StockData.tsx [hunk 5]
```diff
@@ -2,33 +2,36 @@ import style from "./StockData.module.css";
...
+  //   })();
+  // }, []);  
 
   const createRow = (
-    item: { symbol: string; lastPrice: number },
+    item: { symbol: string; price: number },
     index: number
   ) => {
-    return <DataRow key={index} symbol={item.symbol} price={item.lastPrice} />;
+    return <DataRow key={index} symbol={item.symbol} price={item.price} />;
```
Comment (betinabp) [inline TypeScript/AI_Finance/src/Components/StockData/StockData.tsx:26]: Setting indexes as keys is not a good practice. Add `id:number` to `SimulatedStockData ` interface add set `key={id}`

File: TypeScript/AI_Finance/src/index.css [hunk 6]
```diff
@@ -11,4 +11,21 @@
 
 body{
   margin: 0;
-}
\ No newline at end of file
+}
+
+
```
Comment (betinabp) [inline TypeScript/AI_Finance/src/index.css:16]: remove empty line. Make sure you have only one empty line between the css rules

**Review by betinabp — CHANGES_REQUESTED** [review]

**Comment by LuchoStoymirov** [comment]
Header in Stock Data widget

image

**Review by betinabp — APPROVED** [review]

**Pull request merged**